examples/attributes.jl

```diff
@@ -0,0 +1,131 @@
+@block KimFung [attribute] begin
+
+    @cell "algorithm" [vbox, volume, algorithm] begin
+        vbox(
+            volume(rand(32, 32, 32), algorithm = :mip), #with mip algorithm
+            volume(rand(32, 32, 32), algorithm = :absorptionrgba), #with AbsorptionRGBA algorithm
+        )
+    end
+
+    @cell "align" [scatter, text, align] begin
+        scene = Scene()
+        scatter!(scene, rand(10), color=:red)
+        text!(scene,"adding text",textsize = 0.6, align = (:center, :center))
+        display(scene)
+    end
+
+    @cell "color" [lines, color] begin
+        vbox(
+            lines(rand(10), linewidth = 20, color = :blue)
+            lines(rand(10), linewidth = 20, color = to_colormap(:viridis, 10))
```

Comment:
Maybe a comment here about what this does?  You could also just give `color` as `1:10`, which should do the same thing.

Reply:
Done